README.md

```diff
@@ -29,14 +30,19 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 This library works with **Swift 5.2+** , and supports Linux and macOS usage. The minimum macOS version required is **10.15**.
 
-Installation is supported via [Swift Package Manager](https://swift.org/package-manager/).
+### Using in a New Project
```

Comment:
I think the distinction here is more between creating a project from a template and adding it as a dependency manually rather than new vs existing, since users might make a new project without going through the toolbox too. I think something like "Install from template" and "Manual install" may be better section headers. WDYT?

Reply:
yeah, I guess by "existing" what I meant was just "created some other way at some point, either just now or long ago". but that might not be clear. I re-titled the sections to "create a new project from a template" and "add to a project manually"